python/agents/customer-service/tests/unit/test_tools.py

```
@@ -49,9 +46,15 @@ def test_approve_discount():
     result = approve_discount(
         discount_type="percentage", value=10.0, reason="Test discount"
     )
-    assert result == '{"status": "ok"}'
+    assert result == {"status": "ok"}
 
 
+def test_approve_discount():
```

Ruff (0.11.9)

Redefinition of unused `test_approve_discount` from line 45

(F811)